patches/idle-detection.patch

```diff
@@ -0,0 +1,344 @@
+Index: sagemaker-code-editor/vscode/extensions/code-editor-idle-extension/README.md
```

Comment:
nit - keep file name same as extension name `sagemaker-idle-extension.patch` ( unless it includes any additional logic )